src/messages.js

```diff
@@ -136,6 +136,11 @@ function createStructs (overrideTypes) {
       { name: 'userAgent', type: struct.VarString(varint, 'ascii') },
       { name: 'startHeight', type: struct.Int32LE },
       { name: 'relay', type: types.boolean }
+    ]),
+
+    // BIP 0133
+    feefilter: struct([
+      {name: 'feerate', type: types.buffer8}
```

Comment:
Don't mean to nitpick, but I'd like if you added some spacing in the object to keep it consistent with the rest of the file :)

Reply:
Ofc, I was monkey patching locally when playing with the peer network and decided to PR without cleaning up 😛